e2e-tests/insurance/cypress/e2e/journeys/declarations/modern-slavery/modern-slavery-expandable-definition-content.spec.js

```diff
@@ -0,0 +1,199 @@
+import { expandable } from '../../../../../../partials';
+import { PAGES } from '../../../../../../content-strings';
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.DECLARATIONS.MODERN_SLAVERY.EXPANDABLE.VERSIONS[0];
+
+const {
+  ROOT: INSURANCE_ROOT,
+  DECLARATIONS: { MODERN_SLAVERY },
+} = INSURANCE_ROUTES;
+
+const { INTRO, TABLE } = CONTENT_STRINGS;
+
+// TODO: DRY - with anti bribery expandable content
```

Comment:
ticket number?

Reply:
Actually don't think we need this function at all. Updated to using `cy.checkText` instead.